### PR TITLE
chore: up depends

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: apps/management/package.json [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
-    "@ceramicnetwork/http-client": "^1.0.0",
+    "@ceramicnetwork/blockchain-utils-linking": "^1.1.2",
+    "@ceramicnetwork/http-client": "^1.2.0",
     "@ceramicnetwork/transport-postmessage": "^0.3.0",
-    "@ceramicstudio/idx": "^0.12.0",
-    "@ceramicstudio/multiauth": "^0.2.2",
-    "caip": "^0.9.2",
+    "@ceramicstudio/idx": "^0.12.1",
+    "@ceramicstudio/multiauth": "^0.2.3",
+    "caip": "^1.0.0",
```
Comment (oed) [inline apps/management/package.json:24]: This is not a good idea yet afaik! We still use the old caip format.

Comment (oed) [reply apps/management/package.json:24]: 1.0 of caip uses the new caip10 format which ceramic doesn't support yet.

File: packages/connect/package.json [hunk 2]
```diff
@@ -40,25 +40,25 @@
...
+    "@ceramicstudio/idx":"^0.12.1",
+    "@ceramicstudio/idx-tools": "^0.11.0",
     "@ethersproject/hdnode": "^5.0.8",
     "@ethersproject/wallet": "^5.0.11",
     "@types/create-hash": "^1.2.2",
     "@types/store": "^2.0.2",
     "@types/url-parse": "^1.4.3",
-    "caip": "^0.9.2",
-    "dids": "^2.1.0",
+    "caip": "^1.0.0",
```
Comment (oed) [inline packages/connect/package.json:60]: Revert here as well.

File: packages/test-utils/package.json [hunk 3]
```diff
@@ -36,10 +36,10 @@
     "prepublishOnly": "package-check"
   },
   "dependencies": {
-    "@ceramicnetwork/blockchain-utils-linking": "^1.0.0",
+    "@ceramicnetwork/blockchain-utils-linking": "^1.1.2",
     "@ethersproject/hdnode": "^5.0.8",
     "@ethersproject/wallet": "^5.0.11",
-    "caip": "^0.9.2",
+    "caip": "^1.0.0",
```
Comment (oed) [inline packages/test-utils/package.json:42]: Same

**Review by oed — CHANGES_REQUESTED** [review]

File: apps/management/package.json (hunk 1 above)
Comment (oed) [reply apps/management/package.json:24]: See https://github.com/ceramicnetwork/js-ceramic/issues/1657

Comment (oed) [reply apps/management/package.json:24]: See legacy format here: https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md#backwards-compatibility
That's what we're using. 1.0 only supports the new format.

Comment (zachferland) [reply apps/management/package.json]: thanks, rolled that back for now

**Comment by PaulLeCam** [comment]
@zachferland updating IDX solved the previous issues but I got a snapshot mismatch error so I had to update it (https://github.com/ceramicstudio/3id-connect/pull/129/files#diff-a847158362c0fb72d132babe95920bd89469abc2273c91bd518c72b3a81c5371R35). I don't know if that's expected from your changes or it might be surfacing a real issue?

Now the tests pass but there is still an issue with the whole test suite that seems relevant, though as it doesn't seem to be failing in a specific test I don't know what is causing it.

**Comment by oed** [comment]
That snapshot diff is expected @PaulLeCam since we are changing the content of the caip10-link genesis (how the eth address is encoded)

**Comment by zachferland** [comment]
All unit tests pass and integration tests pass as well. 

The test suite fails thought with some request following the ceramic node teardown, not sure what changed to cause that, but  have to look closer at cleanup/teardown now.

Going to merge anyways for now, and look to fix after, so that ci passes again

**Pull request merged**